repos/system_upgrade/el7toel8/actors/rpmscanner/libraries/library.py

```diff
@@ -0,0 +1,12 @@
+
+def get_package_repository_data(self):
```

Comment:
copy&paste error? :)
```suggestion
def get_package_repository_data():
```